### DISCUSSION TITLE
A Firestore Data Converter

### DISCUSSION THREAD

**Discussion by jthoward64** [invertase/react-native-firebase#6514] [Ideas] [answered]
Something similar to the [web SDK's feature](https://firebase.google.com/docs/reference/js/firestore_.documentreference.md#example) would be awesome, especially for typescript users.

**Comment by mikehardy** [comment, marked as answer]
Hey there - duplicate of #5906 - needs work effort though, PRs happily merged

**Reply by jthoward64** [reply]
I knew I had seen it in here before, dunno why search didn't show it; thanks
